chains/evm/contracts/pools/TokenPool.sol

```diff
@@ -1149,32 +1148,19 @@ abstract contract TokenPool is IPoolV2, Ownable2StepMsgSender {
     );
   }
 
-  /// @notice Withdraws all accumulated pool fees to the specified recipient.
-  /// @dev For burn/mint pools, this transfers the entire token balance of the pool contract.
-  /// lock/release pools should override this function with their own accounting mechanism.
-  /// @param recipient The address to receive the withdrawn fees.
-  function withdrawFees(
-    address recipient
-  ) external virtual onlyOwner {
-    uint256 amount = getAccumulatedFees();
-    if (amount > 0) {
-      getToken().safeTransfer(recipient, amount);
-      emit PoolFeeWithdrawn(recipient, amount);
+  /// @notice Withdraws accrued fee token balances to the provided `recipient`.
+  /// @param feeTokens The token addresses to withdraw, including the pool token when applicable.
+  /// @param recipient The address that should receive the withdrawn balances.
```

Comment:
Let's very clearly explain why its OK to transfer out the entire balance of the pool by mentioning lock boxes